tripy/tripy/function_registry.py

```diff
@@ -356,23 +356,54 @@ def raise_overload_error(msg, candidate_overloads, mismatch_reasons=None, extra_
 
         raise_overload_error("Could not find an implementation", self.overloads[key], mismatch_reasons)
 
-    def __call__(self, key: Any):
+    def __call__(self, key: Any, bypass_dispatch: Optional[Union[bool, Sequence[str]]] = None):
         """
         Registers a function with this function registry.
```

Comment:
Can we mention that this works for *documented* methods of classes as well?

Reply:
Added a line to the docstring, good catch.